client/webserver/site/src/js/order.ts

```diff
@@ -192,12 +192,11 @@ export default class OrderPage extends BasePage {
   handleOrderNote (note: OrderNote) {
     const page = this.page
     const order = note.order
+    if (order.id !== this.orderID) return
     this.order = order
     const bttn = page.cancelBttn
-    if (bttn && order.id === this.orderID) {
-      if (bttn && order.status > OrderUtil.StatusBooked) Doc.hide(bttn)
-      page.status.textContent = OrderUtil.statusString(order)
```

Comment:
I don't know why this line would be conditional on `bttn`.

Reply:
Because we're omitting the element in the Go template instead of using `class="d-hide"`. It's a silly way to do it.

Reply:
Oh, so `page.status` doesn't exist unless `page.cancelBttn` exists too?

Reply:
```
      <div class="order-datum">
        <div>[[[Status]]]</div>
        <div>
          <span id="status">{{$ord.StatusString}}</span>
          {{if $ord.Cancelable}}
            <span id="cancelBttn" class="ico-cross fs12 ms-1 pointer hoverbg" data-tooltip="cancel order"></span>
          {{end}
```

Looks like `page.status` would be there even if `page.cancelBttn` is not.

Reply:
I wasn't commenting on why setting the status string was inside of the bttn conditional. Not sure on that, tbh.

Reply:
Gotcha.  I highlighted too many lines with the github comment.  The `page.status` condition must have been an error.